packages/salesforcedx-utils-vscode/src/cli/commandExecutor.ts

```diff
@@ -44,6 +57,34 @@ export class CliCommandExecutor {
       cancellationToken
     );
   }
+
+  protected static patchEnv(
+    options: SpawnOptions,
+    baseEnvironment: Map<string, string>
+  ): SpawnOptions {
+    // start with current process environment
+    const env = Object.create(null);
+
+    // inherit current process environment
+    Object.keys(process.env).forEach(key => {
+      env[key] = process.env[key];
```

Comment:
Can `Object.assign` be used instead of these `forEach` loops?

Reply:
I started with `Object.create(process.env)` but that made the object look weird in the debugger. `Object.assign` works for this one but not for the following one (because it's a map).